File: cpp/level 1/2016.cpp
#include <string>
#include <vector>

using namespace std;

string solution(int a, int b) {
    int mon[12] = {31,29,31,30,31,30,31,31,30,31,30,31};
    string day[7]={"FRI","SAT","SUN","MON","TUE","WED","THU"};
    string answer= "";
	int sum=0;
  
    for(int i=0; i<a-1; i++)
        sum+=mon[i];
    sum += b;
    answer=day[((sum-1)%7)];
	return answer;
}

File: Python/level 1/2016.py
def solution(a, b):
    mon = [31,29,31,30,31,30,31,31,30,31,30,31]
    day = ["FRI","SAT","SUN","MON","TUE","WED","THU"]
    return day[((sum(mon[0:a-1])+b)%7)-1]